python_modules/dagster/dagster_tests/core_tests/asset_defs_tests/test_asset_group.py

```diff
@@ -217,128 +228,195 @@ def _asset_aware():
     return io_manager_obj, _asset_aware
 
 
-def test_asset_group_build_subset_job():
+def _get_assets_defs(use_multi: bool = False):
+    """
+    Dependencies:
+        "upstream": {
```

Comment:
Feel free to disregard, you could avoid the possibility of the implementation getting out of sync with this comment by initializing this dictionary in code and then having a little utility that auto-generates the asset defs from it.

Reply:
smart -- I'll probably bake that into a later portion of this stack to make the merge conflicts less annoying